ds-dl-intro/who_with_quizlink.md

```diff
@@ -1,14 +1,33 @@
-Learners are expected to have the following knowledge:
+This course is for you if:
+
+You have been using machine learning but are starting to dive in to the field of deep learning and you want to learn:
+- What is deep learning?
+- How to structure a deep learning project?
+- What are the different techniques used in deep learning?
+- How to address common pitfalls in a deep learning workflow?
+
+This course assumes that you are currently comfortable with:
 - Basic Python programming skills and familiarity with the Pandas package.
-- Basic knowledge on Machine learning, including the following concepts: Data cleaning, train & test split, type of problems (regression, classification), overfitting & underfitting, metrics (accuracy, recall, etc.).
+- Basic knowledge on Machine learning, including the following concepts: Data cleaning, type of problems (regression, classification), metrics (accuracy, root-mean-square-error, etc.).
+* please note that some relevant concepts will also be covered during the course.
```

Comment:
```suggestion
* please note that some of the traditional machine learning concepts will be covered during the course, since they are also relevant for deep learning.
```